libbpfgo.go

```diff
@@ -630,6 +638,15 @@ func (b *BPFMap) GetValue(key unsafe.Pointer) ([]byte, error) {
 	return value, nil
 }
 
+func (b *BPFMap) GetValue2(key unsafe.Pointer, value *[]byte) error {
```

Comment:
Is it possible to have a better name than `GetValue2`? And also adding a documentation to explain the need of the additional method would be awesome!

Reply:
Hahaha yes, I forgot that I pushed it with that name. I'll update it, thanks for your reviews @kakkoyun !